dataconnect/dataconnect/connector/mutations.gql

```diff
@@ -0,0 +1,113 @@
+# Create a movie based on user input
+mutation createMovie(
+  $title: String!
+  $releaseYear: Int!
+  $genre: String!
+  $rating: Float
+  $description: String
+  $imageUrl: String!
+  $tags: [String!] = []
+) {
+  movie_insert(
+    data: {
+      title: $title
+      releaseYear: $releaseYear
+      genre: $genre
+      rating: $rating
+      description: $description
+      imageUrl: $imageUrl
+      tags: $tags
+    }
+  )
+}
+
+# Update movie information based on the provided ID
+mutation updateMovie(
+  $id: UUID!
+  $title: String
+  $releaseYear: Int
+  $genre: String
+  $rating: Float
+  $description: String
+  $imageUrl: String
+  $tags: [String!] = []
+) {
+  movie_update(
+    id: $id
+    data: {
+      title: $title
+      releaseYear: $releaseYear
+      genre: $genre
+      rating: $rating
+      description: $description
+      imageUrl: $imageUrl
+      tags: $tags
+    }
+  )
+}
+
+# Delete a movie by its ID
+mutation deleteMovie($id: UUID!) {
+  movie_delete(id: $id)
+}
+
+# Delete movies with a rating lower than the specified minimum rating
+mutation deleteUnpopularMovies($minRating: Float!) {
+  movie_deleteMany(where: { rating: { le: $minRating } })
+}
```

Comment:
GQL above don't have `@auth`. 

They aren't accessible by SDK, so would recommend move them either to a `Connector`. Or a README.

Backend probably should reject `UpdateConnector` with operation with `@auth` or at least warn them.